include/fastdds/statistics/dds/domain/DomainParticipant.hpp

```diff
@@ -89,6 +89,13 @@ class DomainParticipant : public eprosima::fastdds::dds::DomainParticipant
             const eprosima::fastdds::dds::DomainParticipant* domain_participant);
 };
 
+/* Environment variable to specify a semicolon-separated list of topic names that define the statistics DataWriters that
+ * the DomainParticipant will enable when created.
```

Comment:
Are the statistics data writers going to be created if the participant is created disabled?

```suggestion
 * the DomainParticipant will enable when enabled.
```

Reply:
Good question! Probably it makes sense that the implementation of the statistics endpoints creation is done only when the participant is enabled.